Makefile

```diff
@@ -628,14 +628,20 @@ $(BUILD_DIR)/src/startup_logo.inc.mio0.o: src/startup_logo.inc.c
 	printf ".include \"macros.inc\"\n\n.data\n\n\n\n.balign 4\n\n\nglabel startup_logo\n\n.incbin \"build/us/src/startup_logo.inc.mio0\"\n\n.balign 16\n\nglabel data_825800_end\n" > build/us/src/startup_logo.inc.mio0.s
 	$(AS) $(ASFLAGS) -o $(BUILD_DIR)/src/startup_logo.inc.mio0.o $(BUILD_DIR)/src/startup_logo.inc.mio0.s
 
-# common_textures.inc.c
-$(BUILD_DIR)/src/common_textures.inc.mio0.o: $(BUILD_DIR)/src/common_textures.inc.o
+LDFLAGS += -R $(BUILD_DIR)/src/common_textures.inc.elf
+
+%/common_textures.inc.elf: %/common_textures.inc.o
+	$(V)$(LD) -t -e 0 -Ttext=0D000000 -Map $@.map -o $@ $< --no-check-sections
+
+%/common_textures.inc.bin: %/common_textures.inc.elf
+	$(V)$(EXTRACT_DATA_FOR_MIO) $< $@
+
+%/common_textures.inc.mio0: %/common_textures.inc.bin
```

Comment:
Since we're beautifying this. Mind adding the sections like so:

```
#==============================================================================#
# Compile Common Textures                                                      #
#==============================================================================#
```

One for Startup logo and one for trophy.

And then delete the `# Compress Segmented Data` one.

Then just after common textures put like (feel free to edit this, I can't think of anything good)
```
#==============================================================================#
# Compile and Link for Final Output                                            #
#==============================================================================#
```

Reply:
Done